packages/asset-service/src/generateAssetData/baseAssets.ts

```diff
@@ -87,7 +87,7 @@ export const osmosis: Asset = {
   name: 'Osmosis',
   precision: 6,
   color: '#750BBB',
-  icon: 'https://assets.coincap.io/assets/icons/osmo@2x.png',
+  icon: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.png',
```

Comment:
nit pick - we can use `githack` here to avoid `429`s from `githubusercontent`- grep for examples

Reply:
👍

Reply:
👍